pkg/volume/cinder/cinder.go

```diff
@@ -227,6 +228,31 @@ func (plugin *cinderPlugin) ConstructVolumeSpec(volumeName, mountPath string) (*
 	return volume.NewSpecFromVolume(cinderVolume), nil
 }
 
+var _ volume.ExpandableVolumePlugin = &cinderPlugin{}
+
+func (plugin *cinderPlugin) ExpandVolumeDevice(spec *volume.Spec, newSize resource.Quantity, oldSize resource.Quantity) (resource.Quantity, error) {
+	cinder, _, err := getVolumeSource(spec)
+	if err != nil {
+		return oldSize, err
+	}
+	cloud, err := plugin.getCloudProvider()
+	if err != nil {
+		return oldSize, err
+	}
+
+	err = cloud.ExpandVolume(cinder.VolumeID, int(newSize.Value()))
```

Comment:
`ExpandVolume`  ideally should take `resource.Quantity` as input and return a `resource.Quantity` as return value which represents new size.

Reply:
I just try to keep this consistent with `CreateVolume` using `int` input. And also `gophercloud` lib will just return err if sth is wrong and nil if everything is ok.

Reply:
I changed that in the end